src/common/paths/query_suggestions.yml

```diff
@@ -2,19 +2,65 @@ get:
   operationId: GetQuerySuggestions
   tags:
     - Query
-  summary: Retrieve query suggestions
+  summary: Retrieve all query functions
+  description: |
+    Returns a list of all query functions.
```

Comment:
What makes these "query functions"? This is new nomenclature.

I feel like the original summary of this endpoint was correct. It returns a limited list of query/function suggestions. I'm just curious how this endpoint is even used. It seems like it's probably specific to the UI implementation to display functions in a specific context. I don't know what they used to determine which functions to return with this endpoint.